awx/main/access.py

```diff
@@ -352,6 +352,9 @@ def get_user_capabilities(self, obj, method_list=[], parent_obj=None, capabiliti
                 # Connot copy manual project without errors
                 user_capabilities[display_method] = False
                 continue
+            elif display_method == 'copy' and (isinstance(obj, Team) or isinstance(obj, User)):
+                user_capabilities[display_method] = False
+                continue
```

Comment:
I think I would only do this for non-superusers, and irrelevant of the method

`elif isinstance(obj, (Team, User)) and (not self.user.is_superuser):`